test/cp.js

```diff
@@ -125,17 +125,23 @@ assert.equal(shell.cat('resources/cp/dir_a/z'), shell.cat('tmp/cp/dir_a/z')); //
 
 //recursive, creates dest dir since it's only one level deep (see Github issue #44)
 shell.rm('-rf', 'tmp/*');
-shell.cp('-r', 'resources/issue44/*', 'tmp/dir2');
+shell.cp('-r', 'resources/issue44', 'tmp/dir2');
```

Comment:
Awesome explanation! The logic sounds right to me. It sounds like this is indeed a bug in the test. Thanks for fixing!